fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1900,8 +1940,10 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 		if (writeBatch == nullptr) {
 			return Void();
 		}
+		TraceEvent("DBEventCommit");
 		auto a = new Writer::CommitAction();
 		a->batchToCommit = std::move(writeBatch);
+		setKeysSet.clear();
```

Comment:
Its safe to clear at this point?

Reply:
Yes.. once the transaction is committed we no more require the keys that are set.